django/contrib/admin/sites.py

```diff
@@ -502,6 +504,12 @@ def app_index(self, request, app_label, extra_context=None):
         ], context)
 
 
+class DefaultAdminSite(LazyObject):
+    def _setup(self):
+        AdminSiteClass = import_string(apps.get_app_config('admin').default_site)
+        self._wrapped = AdminSiteClass()
+
+
 # This global object represents the default admin site, for the common case.
-# You can instantiate AdminSite in your own code to create a custom admin site.
```

Comment:
This comment is still true: one can still instantiate `AdminSite` to create a custom admin site, I don't think this should be removed in case anyone looks here they might only see the `default_site` option.